src/js/videojs.record.js

```diff
@@ -1436,23 +1436,48 @@ class Record extends Plugin {
     }
 
     /**
-     * Create and display snapshot image.
+     * Create snapshot image.
      * @private
      */
     createSnapshot() {
         this.captureFrame().then((result) => {
-            // turn the canvas data into base64 data with a PNG header
-            this.player.recordedData = result.toDataURL('image/png');
+            if (this.player.options_.plugins.record.image.returnType === 'blob')
```

Comment:
@thijstriemstra should I do a check if `.image` is an object? I though know, maybe it could be a boolean?